ci/ci_utest.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I am cool with the ci folder, but maybe this should be under tools/ci?

Reply:
Done